### ISSUE TITLE
Android 14 no Rights to storage needed

### ISSUE THREAD

**Issue by GeorgSommer** [segler-alex/RadioDroid#1224]
I cannot record at all. Running Android 14 there is a popup saying Rights to storage needed. But nowhere I see an option to give that Rights. Neither in the app and neither in Android. I guess the app is still following old Android right behaviour for rights.

_Originally posted by @spardorp in https://github.com/segler-alex/RadioDroid/issues/1213#issuecomment-2156409441_

**Comment by WolfgangAndreas** [comment]
Bump. Same problem here.

**Comment by axel-rank** [comment]
Yes, recording doesn't work. Radiodroid 0.86. Android 14. Galaxy A55, Popup says "Recording needs Writing Permission", but no option to grant writing rights.

**Comment by axel-rank** [comment]
Installed Radiodroid 0.84 from F-Droid. Recording working again.

**Comment by RickWelty** [comment]
Recording didn't work in ver 0.86 (Android 14) per previous comments.  I uninstalled 0.86 and installed ver 0.84 from F-Droid, and recording worked as noted in the previous comment.  Ver 0.84 asked for file modification privilege, unlike ver 0.86 which did not.  I then updated to ver 0.86 using F-droid, and recording continued to work OK.  Maybe the problem is because ver 0.86 does not ask for file modification privilege.  Thanks for a great app !

Update:  Recording works after downgrading to 0.84, then updating to ver 0.86, but a few other things don't work properly - like the track time and storage indicators, and the play/pause button (which starts and stops but does not update the display).  I will go back to ver. 0.84, in which everything seems to work OK.
